src/pudl/analysis/plant_parts_eia.py

```diff
@@ -702,23 +674,14 @@ def add_one_to_many(
         plant_parts_eia_filt = plant_parts_eia.loc[
             plant_parts_eia.record_id_eia.isin(one_to_many.record_id_eia)
         ]
-
-        if plant_parts_eia_filt.empty:  # If 1:m matches not in plant_part subset
-            PLANT_PARTS.pop("plant_match_ferc1")  # Remove from rest of workflow
-            return plant_parts_eia
-
```

Comment:
this pop was the culprit of my lost column woes. I was previously using a function to grab the `id_cols` from `PLANT_PARTS`.

This is not a necessary step at all. I traced the following steps of building the ppe and rebuilt the fast and full version of this ppe and this new assn table.